src/backend/distributed/master/master_stage_protocol.c

```diff
@@ -191,10 +191,14 @@ master_append_table_to_shard(PG_FUNCTION_ARGS)
 	text *sourceTableNameText = PG_GETARG_TEXT_P(1);
 	text *sourceNodeNameText = PG_GETARG_TEXT_P(2);
 	uint32 sourceNodePort = PG_GETARG_UINT32(3);
+
 	char *sourceTableName = text_to_cstring(sourceTableNameText);
 	char *sourceNodeName = text_to_cstring(sourceNodeNameText);
 
-	char *shardName = NULL;
+	Oid shardSchemaOid = 0;
```

Comment:
Can we use similar naming to your PR here;
https://github.com/citusdata/citus/pull/588/files

Such as we can keep shardName, schemaId, schemaName, shardQualifiedName etc.

We would like to have consistency over all the codebase unless old naming does not make sense.